dm/dm/config/source_config.go

```diff
@@ -308,7 +308,7 @@ func (c *SourceConfig) Adjust(ctx context.Context, db *sql.DB) (err error) {
 		log.L().Warn("using an absolute relay path, relay log can't work when starting multiple relay worker")
 	}
 
-	return c.AdjustCaseSensitive(ctx2, db)
+	return nil
```

Comment:
I remembered why this change was introduced, it's because when implementing openAPI , I was referring to the v2 version of the task config 😵